jmx-metrics/src/test/groovy/io/opentelemetry/contrib/jmxmetrics/PrometheusIntegrationTests.groovy

```diff
@@ -41,19 +41,19 @@ class PrometheusIntegrationTests extends IntegrationTest {
         def scraped = receivedMetrics()
 
         then: 'they are of the expected format'
-        scraped.size() == 6
+        scraped.size() == 19
         scraped[0].contains(
                 '# HELP cassandra_storage_load Size, in bytes, of the on disk data size this node manages')
         scraped[1].contains(
-                '# TYPE cassandra_storage_load summary')
+                '# TYPE cassandra_storage_load histogram')
```

Comment:
IIUC, the integration tests seem to just verify the link between SDK and backend without anything particularly JMX specific (we've already verified the mapping to SDK data in unit tests) - we have integration tests in the SDK for this, we could probably remove them from this artifact.